src/lib/functions/registry.js

```diff
@@ -141,6 +143,13 @@ class FunctionsRegistry {
       )
     }
 
+    // This fixes the bug described here https://github.com/netlify/zip-it-and-ship-it/issues/637
+    // If the current function's file is a zip bundle, we ignore it and log a helpful message.
+    if (extname(func.mainFile) === ZIP_EXTENSION) {
+      log(`${NETLIFYDEVWARN} Skipped bundled function. Unzip the archive to load it from source.`)
```

Comment:
I think it would still be useful to show the name of the function.

```suggestion
      log(`${NETLIFYDEVWARN} Skipped bundled function ${chalk.yellow(name)}. Unzip the archive to load it from source.`)
```

Reply:
> I think it would still be useful to show the name of the function.

💯